### ISSUE TITLE
Requested GitHub permissions too greedy

### ISSUE THREAD

**Issue by asgrim** [phpmentoring/testapp#23]
Despite the [AuthController](https://github.com/phpmentoring/testapp/blob/master/src/Mentoring/Controller/AuthController.php#L28) only requesting `['user', 'user:email']` scopes, it seems the permissions phpmentoring is asking for is a little greedy when it comes to organisations, example:

![4a7ec9](https://cloud.githubusercontent.com/assets/496145/7848574/f68712e2-04c4-11e5-8fd0-a88ddbfffa48.png)

This is a no-go for me (and probably many others) as some repositories are private/NDA restricted etc. so it's something we can't do. It may even be that the access is restricted and GitHub's wording is just a little over-zealous, but the way it comes across is that it's asking to access ALL THE THINGS! :smile:

My personal user data I don't really mind about, but I am under NDA for things to do with Roave for example, so at present, I'm blocked from logging in using GitHub account here :crying_cat_face:

**Comment by asgrim** [comment]
FWIW I don't know if it's worth asking whoever the maintainer for https://github.com/thephpleague/oauth2-github is to see if there is an option to turn on/off somewhere?

**Comment by dragonmantank** [comment]
Part of it is Github having crappy wording - when we did ask for organization permissions it was read only, but Github doesn't really specify that. We took that out though.

I think the issue is that asking for more than just public information exposes some of the organization info, and then the wording for Github kicks in saying each organization can specify the access.

This is an issue with the permissions and Github, not with the League package.
